src/robusta/integrations/msteams/msteams_elements/msteams_text_block.py

```diff
@@ -3,34 +3,46 @@
 
 
 class MsTeamsTextBlock(MsTeamsBase):
-    def __init__(self,
-                 text : str,
-                 is_subtle : bool = None,
-                 wrap: bool = None,
-                 weight: str = None,
-                 is_visible : bool = True,
-                 separator : bool = False,
-                 font_size : str = 'medium',
-                 horizontal_alignment : str = "left"
-                 ):
-        super().__init__(self.__text_block(text, is_subtle, wrap, weight, is_visible,
-                                           separator, font_size, horizontal_alignment))
+    def __init__(
+        self,
+        text: str,
+        is_subtle: bool = None,
+        wrap: bool = None,
+        weight: str = None,
+        is_visible: bool = True,
+        separator: bool = False,
+        font_size: str = "medium",
+        horizontal_alignment: str = "left",
+    ):
+        super().__init__(
+            self.__text_block(
+                text,
+                is_subtle,
+                wrap,
+                weight,
+                is_visible,
+                separator,
+                font_size,
+                horizontal_alignment,
+            )
+        )
 
     def __to_msteams_text(self, text: str) -> str:
-        teams_text = MsTeamsMarkDownFixUrl().fix_text(text)
+        teams_text = MsTeamsMarkDownFixUrl().fix_text(str(text))
```

Comment:
This is the actual fix. Sometimes text was a number and it caused an exception down the call stack.